etc/databases/world/updates/updates.sql

```diff
@@ -949,5 +949,36 @@ begin not atomic
+        update creature_template set display_id1 = 1640, display_id2 = 1641, display_id3 = 0, display_id4 = 0 where entry = 1826; -- Scarlet Mage
+        update creature_template set display_id1 = 2467, display_id2 = 2478, display_id3 = 0, display_id4 = 0 where entry = 1827; -- Scarlet Sentinel
+        update creature_template set display_id1 = 1023, display_id2 = 0, display_id3 = 0, display_id4 = 0 where entry = 1831; -- Scarlet Hunter
+        update creature_template set display_id1 = 2039, display_id2 = 0, display_id3 = 0, display_id4 = 0 where entry = 1832; -- Scarlet Magus
+        update creature_template set display_id1 = 1025, display_id2 = 2460, display_id3 = 0, display_id4 = 0 where entry = 1833; -- Scarlet Knight
+        update creature_template set display_id1 = 1025, display_id2 = 2460, display_id3 = 0, display_id4 = 0 where entry = 1834; -- Scarlet Paladin
+        update creature_template set display_id1 = 1640, display_id2 = 1641, display_id3 = 0, display_id4 = 0 where entry = 1835; -- Scarlet Invoker
+        update creature_template set display_id1 = 1025, display_id2 = 2460, display_id3 = 0, display_id4 = 0 where entry = 1836; -- Scarlet Cavalier
+        update creature_template set display_id1 = 2493, display_id2 = 2494, display_id3 = 0, display_id4 = 0 where entry = 1837; -- Scarlet Judge
+        update creature_template set display_id1 = 1611, display_id2 = 1612, display_id3 = 0, display_id4 = 0 where entry = 1838; -- Scarlet Interrogrator
+        update creature_template set display_id1 = 1613, display_id2 = 1614, display_id3 = 0, display_id4 = 0 where entry = 1839; -- Scarlet High Clerist
+        update creature_template set display_id1 = 1641 where entry = 1840; -- Grand Inquisitor
+        update creature_template set display_id1 = 2044 where entry = 1841; -- Scarlet Executioner
+        update creature_template set display_id1 = 1611 where entry = 1843; -- Foreman Jerris
+        update creature_template set display_id1 = 1611 where entry = 1844; -- Foreman Marcrid
+        update creature_template set display_id1 = 1025 where entry = 1845; -- High Protector Tarsen
+        update creature_template set display_id1 = 1025 where entry = 1846; -- High Protector Lorik
+        update creature_template set display_id1 = 1617, display_id2 = 1615, display_id3 = 0, display_id4 = 0 where entry = 1883; -- Scarlet Worker
+        update creature_template set display_id1 = 1025 where entry = 1842; -- Highlord Taelan Fordring
+        update creature_template set display_id1 = 2520, display_id2 = 2521, display_id3 = 0, display_id4 = 0 where entry = 1884; -- Scarlet Lumberjack
+        update creature_template set display_id1 = 2465, display_id2 = 2476, display_id3 = 0, display_id4 = 0 where entry = 1885; -- Scarlet Smith
+
+        -- Remove Tool Bucket spawns from Hearthglen as the associated quest doesn't exist
+        delete from spawns_gameobjects where spawn_entry = 177726;
```

Comment:
Please set as ignored instead of deleting. :pray: (only use delete if it's something we manually spawned like the duplicate NPC spawns from the recent PR).

Reply:
The idea to use ignored instead of deleting came because:

1. This way if we ever find (for some reason) that something was indeed spawned, we don't need to add it from scratch.
2. Our database can be used in the future (by ourselves or by another person) to build a database for newer beta versions or whatever. Unlikely to happen, but you never know!